Remotion/Data/DomainObjects.ObjectBinding.IntegrationTests/SetUpFixture.cs

```diff
@@ -63,6 +63,7 @@ public void OneTimeSetUp ()
 
         var scriptGenerator = new ScriptGenerator(
             pd => pd.Factory.CreateSchemaScriptBuilder(pd),
+            new RdbmsStructuredTypeDefinitionProvider(),
             new RdbmsStorageEntityDefinitionProvider(),
             new ScriptToStringConverter());
```

Comment:
The rationale was that structured types are "more fundamental" than entities, because tables can use structured types in some RDBMS. However, that is not the case for our supported scenarios, so I will change the order.